cloud-chatops/cron-clone.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# When copying the contents of this file, remember to name the new file using only valid characters [a-zA-Z0-9_-]
+# Fullstops '.' and file extenstions are not permitted
+
+set -ex
```

Comment:
Can we use `set -euo pipefail` 
Script will then fail if there are any uninitialised variables or if a pipe fails

Reply:
After having a little read and research about set `-euo pipefail` I agree this is a good error catcher. Alternatively, could we use set `-euo pipefail -x `as this still prints the errors? Not sure if it's overkill for this little script

Reply:
I guess it depends on how noisy it is? Since it's the sort of thing that will run unattended hundreds of times with logs 
I'm fine with either way though